src/data/gridImages.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Mock data for the image list in the overview page.
```

Comment:
It is evident without this comment that this file contains mock data.
Also, the line says this is mock data, but the next line is a type.